testsuite/pytests/test_stdp_nn_synapses.py

```diff
@@ -66,6 +66,18 @@ def setUp(self):
             "tau_minus": 33.7
         }
 
+        # While the random sequences, fairly long, would supposedly
```

Comment:
Could you shorten this comment, just something like "Hard-coded spike sequences testing every possible combination of pre/post precedence"? Although I am not 100% sure what is meant by "precedence" here—just "order"?